### DISCUSSION TITLE
Storable & combinable ammo boxes

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#13199] [Ideas and suggestions]
Two very popular modded QOL additions that have been around for a while I think should be considered for implementation into base game.

- ### Allow ammo boxes and depth charges to be placed into normal crate racks.
They fit the correct dimensions (depth charges if rotated 90 degrees), would allow more flexibility in use of cargo space as well as help supplement certain sub designs which are lacking dedicated ammo storage shelves.

- ### Allow ammo boxes to be merged.
It's literally a one-line XML change. Partial-item results got removed from the game, and now regular deconstruction is off the table as well, so dealing with partial ammo boxes is more annoying than ever. Despite being a hand slot item there seems to be no technical issues whatsoever caused by allowing ammo boxes to be combined.
Also due to requiring hand slot to carry there is not much issue of "gaming the system" by instantly topping off ammo boxes while they are inside the loaders during combat because doing so is hardly any more "overpowered" than just hot-swapping a new ammo box the old way.

**Comment by EtiamNullam** [comment]
These two changes are barely related. The first one might be good, but the other one has already been discussed and it was disliked by the majority (https://github.com/Regalis11/Barotrauma/discussions/12352) so it should be dropped - at least from this discussion.

Especially as you also just revived the that other discussion from over a year ago with a meaningless comment where you lie and call people names.

**Reply by Ek-Videogames** [reply]
'aight have fun hemorrhaging playercount then, if you'll pretend to listen then we'll pretend to care.
